app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,118 @@
+    <header class="c-recipe-view__header">
+      <div class="c-recipe-view__author-box">
+        <img 
+          src=<%= image_url('blank_avatar.jpg') %>
+          class="c-recipe-view__author-img" 
+          alt="recipe author photo" 
+        />
+        <div class="c-recipe-view__author-info">
+          <span class="c-recipe-view__author-name">
+            <%= @recipe.user.name %>
+          </span>
+          <span class="c-recipe-view__date">
+            created <%= time_ago_in_words(@recipe.created_at) %> ago
+          </span>
+        </div>
+      </div>
+
+      <div class="c-recipe-view__action">
+        <%= link_to 'javascript:history.back()', class: "c-icon-btn" do %>
+          <svg class="c-icon--sm">
+            <use xlink:href=<%= image_url("icons/sprite.svg#chevron-left")%>></use>
+          </svg>
+        <% end %>
+        <%= render "components/delete_recipe_btn", recipe: @recipe %>
+      </div>
+    </header>
+
+    <%= render "components/attached_img", 
+      image: @recipe.image, 
+      image_class: "c-recipe-view__img",
+      image_alt: "recipe photo",
+      placeholder: "recipe_placeholder.jpg" 
+    %>
+
+    <h1 class="c-recipe-view__title c-section-title">
+      <%= @recipe.name %>
+    </h1>
+
+    <p class="c-recipe-view__description">
+      <%= @recipe.description %>
+    </p>
+
+    <%= render "components/display_info", infos: %>
+
+    <% if current_user %>
+      <div class="c-recipe-view__btn-group">
+        <%= button_to "generate shopping list", nil,
+          class: "c-btn",
+          remote: true,
+          data: {
+            controller: "popup",
+            action: "popup#openPopup",
+            popup_id_param: "select-inventories-popup"
+          }
+        %>
+        <% if can? :manage, @recipe %>
+          <%= button_to "publish this recipe", nil, class: "c-btn" %>
+        <% end %>
+      </div>
+    <% end %>
+    
+
+    <div class="c-ingredients c-recipe-view__ingredients">
+      <header class="c-ingredients__header">
+        <h2 class="c-ingredients__title">
+          <%= pluralize(@ingredients.length, 'Ingredient') %>
+        </h2>
+        <% if can? :manage, @recipe %>
+          <%= button_to nil,
+            class: "c-icon-btn",
+            remote: true,
+            data: {
+              controller: "popup",
+              action: "popup#openPopup",
+              popup_id_param: "select-food-popup"
+            } do %>
+            <svg class="c-icon--sm">
+              <use xlink:href=<%= image_url("icons/sprite.svg#plus")%>></use>
+            </svg>
+          <% end %>
+        <% end %>
+      </header>
+
+      <% if @ingredients.empty? %>
+        <p>No ingredients added.</p>
+      <% else %>
+        <ul class="c-simple-list">
+          <% @ingredients.each do |ingredient| %>
+            <li>
+              <%= render "components/ingredient_item", ingredient:, recipe: @recipe %>
+            </li>
+          <% end %>
+        </ul>
```

Comment:
[REQUIRED] Kindly include headings for this table just like in the design
<img width="1440" alt="image" src="https://github.com/IvonneBenitesRodriguez/recipe_app/assets/62938735/5e31a2a0-29ab-4359-9bab-ee0929cea8b6">